### DISCUSSION TITLE
New logobumpin

### DISCUSSION THREAD

**Discussion by DzGithub2805** [luckydog7/Funkin-android#1644] [Show and tell]
![Screenshot_20210710-193515_ibisPaint X](https://user-images.githubusercontent.com/83911327/125163157-09399c00-e1b6-11eb-858f-7794642a3985.jpg)
This is what i made

**Comment by zsidebf** [comment]
Download?

**Reply by DzGithub2805** [reply]
Later

**Reply by netonero** [reply]
ah yes later
-yesterday

**Reply by DzGithub2805** [reply]
LOL

**Comment by ghost** [comment]
idk

**Reply by DzGithub2805** [reply]
mm
